src/project-scanner.ts

```diff
@@ -73,21 +75,47 @@ export class ProjectScanner {
     return packageScanner.getConversionManifest();
   }
 
+  /**
+   * Reads the entrypoints for a package from the `main` field in the package's
+   * 'bower.json' file.
+   */
+  private async readBowerJsonEntrypoints(packageName: string) {
+    const bowerJsonPath =
+        this.urlHandler.packageRelativeToOriginalUrl(packageName, 'bower.json');
```

Comment:
This should be `path.join(this.urlHandler.getPackageDir(packageName), 'bower.json')`.

Reply:
Fixed, along with fixing `getPackageDir`.